account/watcher/watcher.go

```diff
@@ -0,0 +1,165 @@
+package watcher
+
+import (
+	"sync"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/chainntnfs"
+)
+
+type WatcherInterface interface { // nolint:golint
```

Comment:
Same here re naming: Just call this `Watcher` and the implementation `watcher`?
Or, together with the other suggestion of extracting the `Handle*` methods into their own interface, this would only handle account expirations. So it could also be called `ExpiryWatcher`?

Reply:
Also: missing godoc and move to `interfaces.go` directly?